mxcube3/routes/Queue.py

```diff
@@ -535,6 +535,7 @@ def add_centring(id):
         409 something bad happened. Plus:
        data ={ "CentringId": newId}
     '''
+    params = request.get_json()
```

Comment:
Perhaps also from pep8 work, params was undefined I assumed that was the request parameters we want. But maybe not ?